source/includes/aws/_volumes.md

```diff
@@ -218,6 +218,38 @@ curl -X DELETE \
 | `taskStatus` <br/>*string* | The status of the operation.                  |
 
 
+<!-------------------- ATTACH A VOLUME -------------------->
+
+#### Attach a volume
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/aws/test-area/volumes/vol-0d1f9106cd0e0dff7?operation=attach"
+```
+
+> Request body example for a volume:
+```json
+{
+    "instanceIdToAttach": "i-0b4e945ee65072b8a",
+    "attachmentDeviceName": "/dev/sda1"
```

Comment:
Can you add some details as to what these request fields are please, you can also specify requirements that the values need to meet for the request to succeed

Reply:
[e7cf94e](https://github.com/cloudops/cloudmc-api-docs/pull/463/commits/e7cf94ef17eece03cc7511592073e167bf293ddf)

![Screen Shot 2022-02-09 at 2 28 11 PM](https://user-images.githubusercontent.com/50206147/153276825-12e21ce1-944e-4ece-92df-2fd507a51dc9.png)